register_types.cpp

```diff
@@ -100,7 +100,7 @@ class ResourceFormatLoaderGDDragonBones : public ResourceFormatLoader
 
 };
 
-static ResourceFormatLoaderGDDragonBones *resource_loader_GDDragonBones = NULL;
+static Ref<ResourceFormatLoaderGDDragonBones> resource_loader_GDDragonBones = NULL;
```

Comment:
This spot is where the shutdown crash was happening; when unloading the engine. This is now set up in the same fashion as demonstrated in the godot documentation

https://docs.godotengine.org/en/stable/development/cpp/custom_resource_format_loaders.html